crates/modelardb_server/tests/integration_test.rs

```diff
@@ -168,6 +168,73 @@ fn test_can_get_schema() {
     stop_modelardbd(flight_server);
 }
 
+#[test]
+#[serial]
+fn test_can_list_actions() {
+    let temp_dir = tempfile::tempdir().expect("Could not create a directory.");
+    let flight_server = start_modelardbd(temp_dir.path());
+
+    let runtime = Runtime::new().expect("Unable to initialize runtime.");
+    let mut flight_service_client = create_apache_arrow_flight_service_client(&runtime, HOST, PORT)
+        .expect("Cannot connect to flight service client.");
```

Comment:
This is a very minor detail but have we made any decisions on what tense to use for error messages? I noticed that we mix present tense `Cannot` and past tense `Could not` quite often. Do you have an opinion on this @skejserjensen?

Reply:
It is honestly not something I have thought about, but I think past tense makes the most sense for errors and log messages as the user reads them after the error has occurred and not while the error is occurring. However, after looking through some of the code I have written I clearly use both tenses interchangeably. What do you think @chrthomsen?

Reply:
Good question... We're not the only ones wondering: https://softwareengineering.stackexchange.com/questions/270899/exception-messages-and-event-logging-which-tense-to-use

I agree that past tense generally is good to use when an attempt was made and it failed. But on the other hand, a message like "File did not exist" also seems weird compared to "File does not exist". So perhaps we should just use the message that most clearly describes the situation?

Reply:
I think I agree with @chrthomsen, that in most cases it should be past tense but there are exceptions like the file one he mentioned (and probably others).

Reply:
I have changed the occurrences of "cannot" in the integration_test.rs file, but I have not gone through the entire project.